dynamic.tex

```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
 </html>
 \end{minted}
 
+
+\begin{aside}{Installing React and ReactDOM}
+
+Now that we've made a project folder using \texttt{npm init},
+and we have a \texttt{project.json} file,
+we can install our first modules:
+
+\begin{minted}{shell}
+npm install react
+npm install reactDOM
+\end{minted}
+
+Any code chunks beginning with the function \texttt{require},
```

Comment:
Um... code chunks? In what? Do you mean calls to `require` in example code snippets?